<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataInputDialog</class>
 <widget class="QDialog" name="dataInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reciever settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="methodSelectionGroupBox">
     <property name="title">
      <string>Method selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="methodUndefinedRadioButton">
        <property name="text">
         <string>Undefined (idle state)</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">methodSelection</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="methodSyntheticRadioButton">
        <property name="text">
         <string>Synthetic</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">methodSelection</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="methodSerialRadioButton">
        <property name="text">
         <string>RS232</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">methodSelection</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="recieverParametersGroupBox">
     <property name="title">
      <string>Reciever parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0" colspan="2">
       <widget class="QWidget" name="recieverSerialWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="recieverSerialGridLayer">
           <item row="0" column="1">
            <widget class="QComboBox" name="recieverSerialComPortComboBox"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="comPortLabel">
             <property name="text">
              <string>COM port</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="recieverSerialBaudRateComboBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="baudrateLabel">
             <property name="text">
              <string>Baudrate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="recieverIdleTimeSpinBox">
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="recieverMaxErrorCountSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="idleTimeLabel">
        <property name="text">
         <string>Idle time [ms]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="maxErrorCountLabel">
        <property name="text">
         <string>Max. error count</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dataInputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dataInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="methodSelection"/>
 </buttongroups>
</ui>
